packages/phrases/src/locales/en/translation/admin-console/general.ts

```diff
@@ -46,6 +46,7 @@ const general = {
   continue: 'Continue',
   page_info: '{{min, number}}-{{max, number}} of {{total, number}}',
   learn_more: 'Learn more',
+  tab_error: '{{count, number}} error',
   tab_errors: '{{count, number}} errors',
```

Comment:
https://www.i18next.com/translation-function/plurals
Use `tab_error_one` and `tab_error_others`

Reply:
This feature is powerful, updated.